### PR TITLE
Path's d updates as native props instead of ui props

### PR DESCRIPTION
**Pull request by osdnk**
Should fx invisible charts

### CODE REVIEW

**Comment by christianbaroni** [comment]
@osdnk haven’t seen invisible charts! but scrubbing is broken, info in header doesn’t change

**Comment by osdnk** [comment]
@christianbaroni  you're tight, but it's good you verified it fixed the issue. I try a bit different approach

**Comment by skylarbarrera** [comment]
Looks like the scrubbing is issue is fixed. 2 🤖 things worth checking out:

Got this crash twice while playing around: http://cloud.skylarbarrera.com/Screenshot_20201222-071921_Rainbow.jpg

Performance on charts seems like it may have taken a hit (only on 🤖 ), any thoughts on this?

File: shim.js [hunk 1]
```diff
@@ -3,9 +3,12 @@ import '@ethersproject/shims';
 import AsyncStorage from '@react-native-community/async-storage';
 // eslint-disable-next-line import/default
 import ReactNative from 'react-native';
+import Animated from 'react-native-reanimated';
 import Storage from 'react-native-storage';
 import logger from 'logger';
 
+Animated.addWhitelistedNativeProps({ d: true });
```
Comment (osdnk) [inline shim.js:10]:
```suggestion
ReactNative.Platform.OS === 'ios' && Animated.addWhitelistedNativeProps({ d: true });
```

**Comment by osdnk** [comment]
Let's for now do it just for iOS

**Review by skylarbarrera — APPROVED** [review]
👍  approving for iOS so we can get a release out

**Review by jinchung — APPROVED** [review]

**Pull request merged**